adafruit_max7219/matrices.py

```diff
@@ -52,16 +52,25 @@ def init_display(self) -> None:
         self.fill(0)
         self.show()
 
-    def text(self, strg: str, xpos: int, ypos: int, bit_value: int = 1) -> None:
+    def text(
+        self,
+        strg: str,
+        xpos: int,
+        ypos: int,
+        bit_value: int = 1,
+        *,
+        font_name: str = "font5x8.bin"
+    ) -> None:
         """
         Draw text in the 8x8 matrix.
 
         :param str strg: string to place in to display
         :param int xpos: x position of LED in matrix
         :param int ypos: y position of LED in matrix
         :param int bit_value: > 1 sets the text, otherwise resets
+        :param str font_name: path to binary font file (default: "font5x8.bin")
```

Comment:
My understanding based on looking how `FrameBuffer.text()` works is that once the font is set for the `Matrix8x8` it can't be changed, and it is set on the first call to this method.  It may be helpful then to add that fact here to prevent future confusion.